uktrade/data-hub-frontend
# Add/Remove company to a list

## Description of change
Trello - https://trello.com/c/ldY9o5Nr/876-add-or-remove-page
The user needs to be able to manage what lists a company features on. The add/remove page will show all lists a user has created and radio options indicating wether that company is saved to the list. By default a company is not added to a list when a list is created. If no lists exist then no options will show so the user will be left with an option to create a list.

## Test instructions
You will need a feature flag set - `companies_add_remove_from_lists` (set this in your local Django admin)
1. Visit a company
2. Click the "Add to my list" button
3. Create a new list

_What should I see?_
 
## Screenshots
![Screenshot 2019-10-08 at 09 48 03](https://user-images.githubusercontent.com/10154302/66381268-e8289380-e9b0-11e9-9c60-ad5e852597cd.png)

![Screenshot 2019-10-08 at 09 49 31](https://user-images.githubusercontent.com/10154302/66381295-f4acec00-e9b0-11e9-9a25-8396609e01bd.png)

Note: **This removes some old tests around adding a company to a single list as this is now legacy.**

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
